### PR TITLE
Introduce Paper

### PR DESCRIPTION
**Pull request by josharsh**
Paper is a cross-platform mobile application that allows scanning documents in real-time.

### CODE REVIEW

**Comment by ritwik12** [comment]
@josharsh We already added YAWP in #2902. I think you forgot to pull.

File: _data/projects/YAWP.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (ritwik12) [inline _data/projects/YAWP.yml]: Already merged in #2902

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/YAWP.yml (hunk 1 above)
Comment (josharsh) [reply _data/projects/YAWP.yml]: Oops

Comment (josharsh) [reply _data/projects/YAWP.yml]: ACK

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@josharsh Thanks!!

**Pull request merged**